tests/test_support/test_support.py

```diff
@@ -155,3 +161,342 @@ def test_refresh_trusted_advisor_check_cycle_continues_on_full_cycle():
     expected_none_response = client.refresh_trusted_advisor_check(checkId=check_name)
 
     expected_none_response["status"]["status"].should.equal("none")
+
+
+@mock_support
+def test_support_case_is_closed():
+    """
+    On closing a case, the correct response is returned
+    """
+    client = boto3.client("support", "us-east-1")
+    create_case_response = client.create_case(
+        subject="test_subject",
+        serviceCode="test_service_code",
+        severityCode="test_severity_code",
+        categoryCode="test_category_code",
+        communicationBody="test_communication_body",
+        ccEmailAddresses=["test_email_cc",],
+        language="test_language",
+        issueType="test_issue_type",
+        attachmentSetId="test_attachment_set_id",
+    )
+    case_id = create_case_response["caseId"]
+
+    resolve_case_response = client.resolve_case(caseId=case_id)
+
+    possible_case_status = [
+        "opened",
+        "pending-customer-action",
+        "reopened",
+        "unassigned",
+        "resolved",
+        "work-in-progress",
+    ]
+    expected_initial_case = [resolve_case_response["initialCaseStatus"]]
+    expected_final_case = "resolved"
+
+    set(expected_initial_case).issubset(
```

Comment:
Assertions should be on separate lines, that makes it easier to debug if it ever fails.
To simplify this:
```
possible_case_status.should.contain(expected_initial_case)
# Expected_final_case has just been set, so it's obvious that this defaults to resolved?
expected_final_case.should.equal("resolved")
```

Reply:
Hey thanks for this. I tried to implement `possible_case_status.should.contain(expected_initial_case)` without subset but struggled to get it to check a `possible_case_status` against `expected_initial_case`. Is there a better way in your experience to achieve this?

Reply:
That doesn't work as a list, you're right. I had to change it to `list.contains(status-as-a-string)` to get it to work.
Made a few more adjustments and used the build-in `pytest.mark.parametrize`, so we don't need an additional dependency. I didn't realize it was there in my initial review, and I figured I might as well change as I'm looking at it anyway. Hope you don't mind :)